### ISSUE TITLE
how to handle multi-class text classification?

### ISSUE THREAD

**Issue by erdenebayrd** [IndicoDataSolutions/finetune#204]
how to handle multi-class text classification problem

**Comment by madisonmay** [comment]
By multiclass do you mean multiple labels are possible per example?

See: https://finetune.indico.io/#finetune.MultiLabelClassifier

It accepts a list of targets as Ys, i.e.:

```python
model = MultiLabelClassifier()
model.fit(X, [['class1', 'class2'], ['class1'], ['class3', 'class1'], ...])
```
